modules/ztp-setting-managed-bare-metal-host-kernel-arguments-gitops-workflow.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-arguments_{context}"]
+= Setting managed bare-metal host kernel arguments
+
+Specify kernel arguments for the Discovery ISO to customize the boot process in managed bare-metal hosts. This is useful for clusters with specific environmental requirements. For example, configure the `rd.net.timeout.carrier` kernel argument to facilitate static networking or to receive a DHCP address before downloading the root file system. 
+
+[NOTE]
+====
+In {product-title} {product-version}, you can only add kernel arguments. You can not replace or delete kernel arguments.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have logged in to the hub cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create the `InfraEnv` CR and edit the `spec.kernelArguments` specification to configure kernel arguments.
+
+.. Save the following YAML in an `InfraEnv-example.yaml` file:
++
+[NOTE]
+====
+The `InfraEnv` CR has template variables such as `{{ .Cluster.ClusterName }}`. The `SiteConfig` CR automatically populates values for these templates during deployment. Do not edit the templates manually.  
+====
++
+[source,yaml,options="nowrap",role="white-space-pre"]
+----
+apiVersion: agent-install.openshift.io/v1beta1
+kind: InfraEnv
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
+  name: "{{ .Cluster.ClusterName }}"
```

Comment:
Can we elaborate on where these template variables get populated from from?

Reply:
Ah they get populated from the SiteConfig in the same folder. @imiller0 is that right?